functionsFix cannot install firmware on 11TC50Q/11TQ50Q

wodnd999999 opened the pull request:
This commit fix can't install firmware on LG Chromebook 11TC50Q/11TQ50Q after a9664c commit.
In the a9664c commit, the device variable was rewritten to boardName, making it impossible to install firmware.
It was solved by modifying the value to fit 11TC50Q/11TQ50Q.

MrChromebox reviewed (commented):
we don't want to change, we want to support both. There's probably a few other JSL boards that need to be added as well

wodnd999999 commented:
Sorry for my poor English.

As I understand it, your comment seems to want to include both the original and modified ones to support other boards as well. However, the reason I modified the code is because it became impossible to install on the same device. I suggested adding support for that device in #192.

This fix is ​​not caused by other revision boards of the same model, but is believed to be a bug caused by the recently fixed code. Previously, the device variable was imported to the following code and 'kracko' was returned.
`device=$(dmidecode -s system-product-name | tr '[:upper:]' '[:lower:]' | sed 's/ /_/g' | awk 'NR==1{print $1}')`

On the other hand, in the [recent commit](https://github.com/MrChromebox/scripts/commit/a9664c4e138a18d29bd49579343f92470aa1f692), the following code was added and returned 'kracko360' from the exact same device, so it is impossible to install as a result.
`device=${boardName,,}`

MrChromebox commented:
should be fixed now
